tsconfig.json

```diff
@@ -1,6 +1,8 @@
 {
   "references": [
     { "path": "packages/compiler/tsconfig.json" },
+    { "path": "packages/versioning/tsconfig.json" },
+    { "path": "packages/azure-functions/tsconfig.json" },
```

Comment:
azure-functions package doesn't seem to exist and Ctrl+Shift+B in VS Code fails

Reply:
ahh yeah whoops, that's just locally for experimentation. Will delete.